Module2-AspNetCore/README.md

```diff
@@ -376,11 +376,22 @@ In this task you'll move the middleware to a separated file.
 	}
 	```
 
-1. Back in the application's **Startup.cs** file, replace the inline middleware delegate with the call to the `app.UseRequestCulture()` method to add your new middleware class to the HTTP pipeline.
+1. Back in the application's **Startup.cs** file, replace the inline middleware delegate with the call to the `app.UseRequestCulture()` method to add your new middleware class to the HTTP pipeline. When you're done, your `Configure` method should look like the code below:
```

Comment:
When I first followed I replaced more lines than I should (deleted app.Run() and ran into HTTP 404 error when debugging it). Better to show exactly what it looks like here